build_your_own_images.md

```diff
@@ -68,17 +68,11 @@ ENV KONG_VERSION $KONG_VERSION
 
 # Uncomment the ARG KONG_SHA256 line to build a container using a .deb or .rpm package
 # For .deb packages, the SHA is in
-# https://download.konghq.com/gateway-<gateway-major-version>-<os>-<os_version>/dists/default/all/binary-amd64/Packages
+# https://cloudsmith.io/~kong/repos/gateway-<gateway-major-version><gateway-minor-version>/packages/detail/deb/kong/<gateway-version>/a=amd64;xc=main;d=debian%252F<os_version>;t=binary/
 # For .rpm packages, the SHA is in
-# https://download.konghq.com/gateway-<gateway-major-version>-<os>-<os_version>/repodata/<some-sha>-primary.xml.gz
+# https://cloudsmith.io/~kong/repos/gateway-<gateway-major-version><gateway-minor-version>/packages/detail/rpm/kong/<gateway-version>/a=x86_64;d=el%252F<os_version>;t=binary/
 # ARG KONG_SHA256="<.deb-or.rpm-SHA>"
 
-# Uncomment to build a container using the .deb.tar.gz Kong Gateway package
-# For .deb.tar.gz packages, the SHA is in
-# https://download.konghq.com/gateway-<gateway-major-version>-ubuntu/PULP_MANIFEST
-# ARG KONG_AMD64_SHA="<amd64_sha>"
-# ARG KONG_ARM64_SHA="<arm64_sha>"
```

Comment:
I removed the mentions to this `.deb.tar.gz` packages as I could not find them. If the reference is still relevant, please let me know and I will add it back.

Reply:
Agreed with this, leaving the comment up and "unresolved" for future reviewers.